app/src/main/res/values/strings.xml

```diff
@@ -31,4 +31,6 @@
     <string name="profile_cancel">Cancel</string>
     <string name="profile_pic_upload_text">Profile Picture Upload</string>
     <string name="profile_pic_text">Profile Picture</string>
+    <string name="settings_text">Settings</string>
+    <string name="log_out_text">Log Out</string>
```

Comment:
We have second string constant for "Logout" above. I'd say just replace that one with "Log Out" if you think it looks better.